projects/demo/src/modules/components/editor/images/upload/examples/1/index.ts

```diff
@@ -52,17 +55,29 @@ import {ImgbbService} from './imgbb.service';
     changeDetection,
     encapsulation,
 })
-export class TuiEditorUploadingImagesExample1 {
+export class TuiEditorUploadingImagesExample1 implements OnInit {
+    @ViewChild(TuiEditorComponent, {static: true})
+    private readonly wysiwyg!: TuiEditorComponent;
+
     readonly builtInTools = [TuiEditorTool.Undo, TuiEditorTool.Img];
 
     control = new FormControl('');
 
-    constructor(@Inject(ImgbbService) readonly imgbbService: ImgbbService) {
+    constructor(
+        @Inject(DOCUMENT) readonly documentRef: Document,
+        @Inject(ImgbbService) readonly imgbbService: ImgbbService,
+    ) {
         this.control.patchValue(
             '<img data-type="image-editor" src="/assets/images/lumberjack.png" width="300">' +
                 '<p>Try to drag right border of image!</p>' +
                 '<p>To change min/max size of image use token <code>TUI_IMAGE_EDITOR_OPTIONS</code>.</p>' +
                 '<img src="data:image/svg+xml;base64,PHN2ZyB2ZXJzaW9uPSIxLjIiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmlld0JveD0iMCAwIDUwIDUwIiB3aWR0aD0iNTAiIGhlaWdodD0iNTAiPjxzdHlsZT4uYXtmaWxsOiNkZDAwMzF9LmJ7ZmlsbDojYzMwMDJmfS5je2ZpbGw6I2ZmZn08L3N0eWxlPjxwYXRoIGNsYXNzPSJhIiBkPSJtNDMuNiAxMi42bC0yLjggMjQuNy0xNS44IDguNy0xNS44LTguNy0yLjgtMjQuNyAxOC42LTYuNnoiLz48cGF0aCBjbGFzcz0iYiIgZD0ibTI1IDZsMTguNiA2LjYtMi44IDI0LjctMTUuOCA4Ljd2LTE1LjMtMjAuMy00LjR6Ii8+PHBhdGggZmlsbC1ydWxlPSJldmVub2RkIiBjbGFzcz0iYyIgZD0ibTM2LjYgMzYuNWgtNC4zbC0yLjQtNS44aC05LjlsLTIuMyA1LjhoLTQuM2wxMS42LTI2LjF6bS0xMS42LTE3LjZsLTMuNCA4LjJoNi44eiIvPjwvc3ZnPg==" />',
         );
     }
+
+    ngOnInit(): void {
```

Comment:
Maybe we can use `tuiValidator` directive for more declarative approach?

Reply:
fixed